src/matchers/toHaveBeenCalledOnceWith.js

```diff
@@ -1,6 +1,6 @@
 import { isJestMockOrSpy } from '../utils';
 
-export function toHaveBeenCalledOnceWith(received, expected) {
+export function toHaveBeenCalledOnceWith(received, ...expected) {
```

Comment:
As [mentioned](https://github.com/jest-community/jest-extended/issues/518#issuecomment-1409835822) this PR should probably also change the types to match the new implementation

Reply:
Yea, I realized I missed that at first. Thank you. Fixed.